.gitignore

```diff
@@ -1,3 +1,4 @@
 build
 composer.lock
-vendor
\ No newline at end of file
+vendor
+.DS_STORE
```

Comment:
Why would one **not** want .DS_STORE files ignored? A .DS_STORE file "is a hidden file with a proprietary format created by OS X to store custom attributes of a folder such as the position of icons or the choice of a background image... these files were primarily used by the Finder [Mac application]"

I always add hidden temporary/application-irrelevant files to the .gitignore files of my projects.